DotaApiCore/HeroDetails/Models/Hero.cs

```diff
@@ -5,12 +5,12 @@ namespace DotaApiCore.HeroDetails.Models
     public class Hero
     {
         [JsonProperty("name")]
-        public string Name { get; set; }
+        public string HeroName { get; set; }
 
         [JsonProperty("id")]
         public int HeroId { get; set; }
 
         [JsonProperty("localized_name")]
-        public string LocalizedName { get; set; }
+        public string HeroLocalizedName { get; set; }
```

Comment:
I don't agree with this, `Hero.HeroName` or ` Hero.HeroLocalizedName` is much too verbose